### PR TITLE
End of Year: add strings

### PR DESCRIPTION
**Pull request by leandroalonso**
Adds strings needed for the upcoming EoY.

The reason to add those strings right now is so we're able to have enough time to review them in advance.

## To test

1. Green CI is enough
2. Just review the descriptions and strings

## Checklist

- [x] I have considered if this change warrants user-facing release notes and have added them to `CHANGELOG.md` if necessary.
- [x] I have considered adding unit tests for my changes.
- [x] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.

### CODE REVIEW

File: podcasts/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -3293,3 +3296,48 @@
...
+"eoy_story_longest_episode" = "The longest episode you listened to was %1$@ from the podcast %2$@";
+
+/* Subtitle for the story showing the longest episode listened for the user in the current year. %1$@ is a placeholder for the episode length. */
+"eoy_story_longest_episode_duration" = "This episode was %1$@ long";
+
+/* Title for the epilogue story */
+"eoy_story_epilogue_title" = "Thank you for letting Pocket Casts be a part of your listening experience in 2022";
+
+/* Subtitle for the epilogue story */
+"eoy_story_epilogue_subtitle" = "Don't forget to share with your friends and give a shout out to your favorite podcasts creators";
```
Comment (emilylaguna) [inline podcasts/en.lproj/Localizable.strings]: Podcast creators makes more sense here, right?
```suggestion
"eoy_story_epilogue_subtitle" = "Don't forget to share with your friends and give a shout out to your favorite podcast creators";
```

File: podcasts/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -3276,6 +3276,9 @@
 /* Title for the End of Year feature */
 "eoy_title" = "Your Year in Podcasts";
 
+/* Title for the End of Year feature */
+"eoy_title" = "Your Year in Podcasts";
+
```
Comment (emilylaguna) [inline podcasts/en.lproj/Localizable.strings]: This is duplicated above.

**Review by emilylaguna — CHANGES_REQUESTED** [review]
Left some suggestions below.

File: podcasts/en.lproj/Localizable.strings (hunk 2 above)
Comment (leandroalonso) [reply podcasts/en.lproj/Localizable.strings]: Ops. Done in ba310d5c7a53c5397dfa11c17ff75c24b64adc86

**Review by emilylaguna — APPROVED** [review]

**Pull request merged**